apps/users/fields.py

```diff
@@ -14,8 +14,8 @@ class CommaSeparatedEmailField(Field):
     )]
 
     widget = Input(attrs={
-        'placeholder': 'jana@beispiel.de, tobias@beispiel.de, '
-                       'swetlana@beispiel.de,…'
+        'placeholder': 'maria@beispiel.de, peter@beispiel.de, '
+                       'nicola@beispiel.de,…'
```

Comment:
We might wanto to use a [reserved domain](https://tools.ietf.org/html/rfc2606) like `example.com`.